apps/bfd-server/dev/api-changelog.md

```diff
@@ -1,5 +1,20 @@
 # API Changelog
 
+## BFD-1916: Map FI Claim Action Code in V1 and V2
+
+FI Claim Action Code data existed in our code but was not mapped for V1 or V2. This
+data is now available in the extensions for SNF and Inpatient claims:
+```json
+  {
+  "url" : "https://bluebutton.cms.gov/resources/variables/fi_clm_actn_cd",
+  "valueCoding" : {
+    "system" : "https://bluebutton.cms.gov/resources/variables/fi_clm_actn_cd",
+    "code" : "1",
+    "display" : "Original debit action (always a 1 for all regular bills)"
+  }
+}
```

Comment:
In the below examples we were showing a larger snippet that showed the hierarchy starting from the top; do we want to keep this consistent and start with resource (to show this is in EOB and where it is)? Mentioning the path as described in the story here may also be useful to users.

Reply:
Updated